guide/message_transformation/message_transformation.bal

```diff
@@ -0,0 +1,411 @@
+
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/io;
+import ballerina/log;
+import ballerina/mysql;
+//import ballerinax/kubernetes;
+//import ballerinax/docker;
+
+//connect the student details table
+endpoint mysql:Client StudentDetailsDB {
```

Comment:
Conventionally, we use camel case for endpoint names. Better if we can consistently stick to that format, especially since this is coming from the org itself.